movement/watch_faces/complication/tarot_face.c

```diff
@@ -103,12 +120,22 @@ static void tarot_display(tarot_state_t *state) {
     }
 
     // show a special status if we're looking at the first or last card in the spread
-    if (state->current_card == 0) {
-        start_end_string = "St";
-    } else if (state->current_card == state->num_cards_to_draw - 1) {
-        start_end_string = "En";
-    } else {
-        start_end_string = "  ";
+    start_end_string = "  ";
+    watch_clear_indicator(WATCH_INDICATOR_LAP);
+    if (TAROT_LOOP_STYLE != TAROT_LOOP_STYLE_NONE && state->num_cards_to_draw > 1) {
```

Comment:
if you want to keep the ability to configure this at compile time, change these `if`'s to `#ifdef` statements to avoid unnecessary code making its way into the compiled object.